src/components/App/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import Movie from '../Movies/Movies';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [
```

Comment:
We should pull the data filw in before we move on to the next iteration to make sure everything is populating correctly!